dv/uvm/icache/doc/ibex_icache_dv_plan.md

```diff
@@ -1,99 +1,142 @@
 ---
-title: "IBEX_ICACHE DV Plan"
+title: "Ibex ICache DV Plan"
 ---
 
 ## Goals
 * **DV**
-  * Verify all IBEX_ICACHE IP features by running dynamic simulations with a SV/UVM based testbench
-  * Develop and run all tests based on the [testplan](#testplan) below towards closing code and functional coverage on the IP and all of its sub-modules
+  * Verify all Ibex ICache IP features by running dynamic simulations with a SV/UVM based testbench
+  * Develop and run all tests based on the [testplan](#testplan) below towards closing code and functional coverage.
 * **FPV**
-
-## Current status
-<!-- TODO for now -->
-* [Design & verification stage]({{< relref "doc/project/hw_dashboard" >}})
-  * [HW development stages]({{< relref "doc/project/hw_stages" >}})
-* [Simulation results](https://reports.opentitan.org/hw/ip/ibex_icache/dv/latest/results.html)
```

Comment:
we're going to want to keep this link in this dv plan, as once you start running regressions with dvsim, the sim results (and coverage too!) will be made publicly visible by being uploaded to this link at the opentitan web server.

Reply:
As above. Yes, I agree that we need to figure out how to publish these results from a different repository from OpenTitan, but that's surely a different problem from "write documentation explaining how we're testing the icache", which this PR seeks to address.

Reply:
Would you at least retain the header with TBD instead of actual links?

Reply:
Well, the links won't work (because `relref` doesn't work outside of the OT repo). Would you like me to re-instate the auto-generated bullet points?

Reply:
Hence the TBDs.

Reply:
Ok. Will do as instructed.